contracts/KyberOasisReserve.sol

```diff
@@ -0,0 +1,258 @@
+pragma solidity 0.4.18;
+
+
+import "./ERC20Interface.sol";
+import "./Utils.sol";
+import "./Withdrawable.sol";
+import "./KyberReserveInterface.sol";
+
```

Comment:
I need some background to review this...

Reply:
talked f2f.